iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractTapChangerTest.java

```diff
@@ -199,6 +199,92 @@ public void testDefaultPhaseTapChangerStep() {
 
+    @Test
+    public void testPhaseTapChangerStepsReplacer() {
+        PhaseTapChanger phaseTapChanger = twt.newPhaseTapChanger()
+                .setTapPosition(1)
+                .setLowTapPosition(0)
+                .setRegulating(true)
+                .setTargetDeadband(1.0)
+                .setRegulationMode(PhaseTapChanger.RegulationMode.ACTIVE_POWER_CONTROL)
+                .setRegulationValue(10.0)
+                .setRegulationTerminal(terminal)
+                .beginStep()
+                .setR(1.0)
+                .setX(2.0)
+                .setG(3.0)
+                .setB(4.0)
+                .setAlpha(0.0)
+                .setRho(1.0)
+                .endStep()
+                .beginStep()
+                .setR(1.0)
+                .setX(2.0)
+                .setG(3.0)
+                .setB(4.0)
+                .setAlpha(5.0)
+                .setRho(6.0)
+                .endStep()
+                .add();
+        assertEquals(2, phaseTapChanger.getStepCount());
+        assertEquals(2, phaseTapChanger.getAllSteps().size());
+        assertEquals(0, phaseTapChanger.getLowTapPosition());
+        assertEquals(1, phaseTapChanger.getHighTapPosition());
+        assertEquals(0, phaseTapChanger.getNeutralPosition().orElseThrow());
+
+        //check neutral step attributes
+        PhaseTapChangerStep neutralStep = phaseTapChanger.getNeutralStep().orElseThrow();
+        assertEquals(0, neutralStep.getAlpha());
+        assertEquals(1, neutralStep.getRho());
+        assertEquals(1, neutralStep.getR());
+        assertEquals(2, neutralStep.getX());
+        assertEquals(3, neutralStep.getG());
+        assertEquals(4, neutralStep.getB());
+
+        //replace steps
+        phaseTapChanger.stepsReplacer()
+                .beginStep()
+                .setR(1.0)
+                .setX(2.0)
+                .setG(3.0)
+                .setB(4.0)
+                .setAlpha(5.0)
+                .setRho(6.0)
+                .endStep()
+                .beginStep()
+                .setR(5.0)
+                .setX(6.0)
+                .setG(7.0)
+                .setB(8.0)
+                .setAlpha(6.0)
+                .setRho(7.0)
+                .endStep()
+                .beginStep()
+                .setR(9.0)
+                .setX(10.0)
+                .setG(11.0)
+                .setB(12.0)
+                .setAlpha(0.0)
+                .setRho(1.0)
+                .endStep()
+                .replaceSteps();
+
+        assertEquals(3, phaseTapChanger.getStepCount());
+        assertEquals(3, phaseTapChanger.getAllSteps().size());
+        assertEquals(0, phaseTapChanger.getLowTapPosition());
+        assertEquals(2, phaseTapChanger.getHighTapPosition());
+        assertEquals(2, phaseTapChanger.getNeutralPosition().orElseThrow());
+
+        //check neutral step attributes
+        neutralStep = phaseTapChanger.getNeutralStep().orElseThrow();
+        assertEquals(0, neutralStep.getAlpha());
```

Comment:
Could you call the `assertEquals(...)` method with the tolerance argument (3 args signature) when comparing doubles?